eng/stages/custom-publish.yml

```diff
@@ -2,128 +2,44 @@ parameters:
   dependsOnPublishStages: []
 
 stages:
-# Create one stage per pipeline that needs some extra publish steps.
+
+# Stages-based publishing entry point
+- template: /eng/common/templates/post-build/post-build.yml
+  parameters:
+    dependsOn:
+    - ${{ each dependency in parameters.dependsOnPublishStages }}:
+      - CustomPublishBlob_${{ dependency.dependsOn }}
+    # Symbol validation is not ready yet. https://github.com/dotnet/arcade/issues/2871
+    enableSymbolValidation: false
+    # Doesn't work yet, not fully configured. https://github.com/dotnet/core-setup/issues/5254
+    enableSigningValidation: false
+    # SourceLink validation doesn't work in dev builds: tries to pull from GitHub. https://github.com/dotnet/arcade/issues/3604
+    enableSourceLinkValidation: false
+    # Allow symbol publish to emit expected warnings without failing the build. Include single
+    # quotes inside the string so that it passes through to MSBuild without script interference.
+    symbolPublishingAdditionalParameters: "'-warnAsError:$false'"
+
+# Create extra stages, per BAR channel that needs some extra publish steps.
 - ${{ each dependency in parameters.dependsOnPublishStages }}:
-  - stage: CustomPublish_${{ dependency.dependsOn }}
-    displayName: 'Custom: ${{ dependency.channel.name }}'
+  - stage: CustomPublishBlob_${{ dependency.dependsOn }}
```

Comment:
Here is another place where I think the `Customer` prefix is adding more confusion for me.  It is the only place this channel is published to blob storage isn't it?  In general it would help the readability if the 'Customer' publish stages/jobs were named simply in terms of what they are publishing without the prefix.